node/runtime/pangolin/src/pallets/transaction_pause.rs

```diff
@@ -0,0 +1,9 @@
+// --- darwinia-network ---
+use crate::*;
+use module_transaction_pause::Config;
+
+impl Config for Runtime {
+	type Event = Event;
+	type UpdateOrigin = ApproveOrigin;
```

Comment:
It is ok for Pangoro and Pangolin, since the root key can be used as the UpdateOrigin or Operation key, but for chains without Root, or Root is multisig, it is better to set a separate and Operation Key Set(Using Oneof), for emergency operational consideration.

Reply:
Just for testing.

Darwinia/Crab/Acala use `2/3` council as `ApproveOrigin`

Reply:
https://github.com/darwinia-network/darwinia/issues/776